cypress/support/config/envs.js

```diff
@@ -21,6 +21,17 @@ const config = {
     standaloneErrorPages: false,
     alwaysCheckForFallback: true,
   },
+  stage: {
+    baseUrl: 'https://www.stage.bbc.com',
+    dataUrl: 'https://www.stage.bbc.com',
+    assetUrl: 'https://news.stage.files.bbci.co.uk/include/articles/public',
+    assetOrigin: 'https://news.stage.files.bbci.co.uk',
```

Comment:
As part of our CD pipeline, we deploy our static assets to test and live `https://news.test.files.bbci.co.uk` & `https://news.
.files.bbci.co.uk`, but not stage. I am not sure whether this stage endpoint will have any of our assets available.

If you want stage to mimick test, should this and the assetUrl above not have this domain `https://news.test.files.bbci.co.uk`?

Reply:
I honestly didnt know the complete differences between stage and other environments. Ill fix that then.